db/v-2/insert-test-data/db.changelog-insert.xml

```diff
@@ -7,6 +7,7 @@
 
     <include file="user-insert.xml" relativeToChangelogFile="true"/>
     <include file="organization-insert.xml" relativeToChangelogFile="true"/>
+    <include file="test_suites_source-insert.xml" relativeToChangelogFile="true"/>
```

Comment:
Let's use as a separator in the file name only _-_
_test-suites-source-insert.xml_

Reply:
idea was to use format `<table_name>-insert.xml` similar to liquibase change logs for tables: `test_suite.xml` and `test_execution.xml`.
But as I can see, we use both formats: `lnk-contest-execution.xml`, `lnk-contest-project.xml` and etc.

Do I need to rename only this file or also `test_suites_source.xml`?

Reply:
I don't think it's essential, but I would prefer using the hyphen "-" because it doesn't require additional keystrokes on the keyboard.
It's probably not worth renaming old files anymore, so as not to break the changeSet.

Reply:
since `-` is more preferable, named `test_suites_source.xml` and `test_suites_source-insert.xml`